templates/experiments/single_experiment_home.html

```diff
@@ -212,54 +212,244 @@ <h3 class="font-bold text-lg">Parent Experiments:
+                      <!-- Where/AND Label -->
+                        <div class="text-gray-500 font-medium w-16">
+                          <span x-text="index === 0 ? 'Where' : 'AND'"></span>
+                        </div>
+                      <!-- Column Selector -->
+                        <div class="relative py-2">
+                          <div class="dropdown dropdown-bottom">
+                            <label
+                              tabindex="0"
+                              class="block w-40 px-3 border border-gray-300 rounded-lg bg-white text-sm cursor-pointer flex justify-between items-center"
+                              style="height: 2rem;"
+                            >
+                              <span x-text="filter.column ? filterData.columns[filter.column].label : 'Column'"></span>
+                              <i class="fa-solid fa-caret-down fa-sm"></i>
+                            </label>
+                            <ul tabindex="0" class="dropdown-content z-[1] menu p-2 bg-base-100 rounded-box border border-gray-300 w-40">
+                              <li><a @click="filter.column = ''; updateOperatorsByColumn(index)">Column</a></li>
+                              <template x-for="(config, column) in filterData.columns" :key="column">
+                                <li><a @click="filter.column = column; updateOperatorsByColumn(index)" x-text="config.label"></a></li>
+                              </template>
+                            </ul>
+                          </div>
+                        </div>
+                      <!-- Operator Selector -->
+                        <div x-show="filter.column" class="relative py-1 px-1">
+                          <div class="dropdown dropdown-bottom">
+                            <label
+                              tabindex="0"
+                              class="block w-40 px-3 border border-gray-300 rounded-lg bg-white text-sm cursor-pointer flex justify-between items-center"
+                              style="height: 2rem;"
+                            >
+                              <span x-text="filter.operator || 'Select operator'"></span>
+                              <i class="fa-solid fa-caret-down fa-sm"></i>
+                            </label>
+                            <ul tabindex="0" class="dropdown-content z-[1] menu p-2 bg-base-100 rounded-box border border-gray-300 w-40">
+                              <template x-for="operator in filter.availableOperators" :key="operator">
+                                <li><a @click="filter.operator = operator; triggerFilterChange()" x-text="operator"></a></li>
+                              </template>
+                            </ul>
+                          </div>
+                        </div>
+                        <template x-if="filter.operator">
+                          <div class="relative py-1 pl-1">
```

Comment:
same as above re spacing
```suggestion
                          <div class="relative">
```